defaults/main.yml

```diff
@@ -4,23 +4,26 @@
 # Configuration settings for the daemon.
 dhcpd_ipv4_interface: "{{ ansible_default_ipv4.interface | default(eth0) }}"
 
-# Setting applicable for the global scope.
-dhcpd_default_lease_time: 600
-dhcpd_max_lease_time: 7200
-dhcpd_subnet_mask: 255.255.255.0
-dhcpd_broadcast_address: 10.0.2.255
-dhcpd_routers: 10.0.2.254
-dhcpd_domain_name_servers: 192.168.1.1, 192.168.1.2
-dhcpd_domain_search: example.com
+# DEFAULT statements applicable global if not overwritten at subnet level
+# can be any valid DHCP ISC statements https://kb.isc.org/docs/isc-dhcp-41-manual-pages-dhcpdconf#
+# please read carefully docs when changing!!!!
+# e.g. domain-search DHCP requires explicitly to enclose in " as a string
 
-# The image to serve for PXE booting.
-dhcpd_filename: pxelinux.0
-# Where the image can be downloaded from.
-dhcpd_next_server: 10.0.2.254
+dhcpd_global_statements:
+  parameters:
```

Comment:
I'm not a fan of dictionaries for a set of settings, it would require a user of this role to paste the full dictionary in order to modify a single value.

Image I want to use this role and change for example the `default-lease-time`, I'd have to rewrite the whole dict:

```yaml
dhcpd_global_statements:
￼  parameters:
    default-lease-time: 1234 # <- this is the only change.
￼    max-lease-time: 7200
￼  options:
￼    domain-name-servers: "192.168.1.1, 192.168.1.2"
￼    domain-search: '"example.com"'
```